src/models/customer.js

```diff
@@ -0,0 +1,37 @@
+const prisma = require("../utils/prisma");
+
+const customer_contact = {
+  select: {
+    id: true,
+    name: true,
+    createdAt: true,
+    updatedAt: true,
+    contact: {
+      select: {
+        id: true,
+        customerId: true,
+        phone: true,
+        email: true,
+        createdAt: true,
+        updatedAt: true,
+      },
+    },
+  },
+};
```

Comment:
could you explain how setting out the contact info like this works? Is it defining each parameter as existing? I'm unsure as to the use of this?